articles/resources/relnotes.md

```diff
@@ -17,6 +17,16 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version X.XX.XXXXXXXXXX
+
+*Release date: January 26th, 2021*
+
+- Simplified qubit allocation, providing more convenient syntax for allocating qubits, [see details in Q# language repository](https://github.com/microsoft/qsharp-language/blob/main/Approved/1-implicitly-scoped-qubit-allocation.md).
+- Improved the chemistry library including API improvements and better Python support and integration with existing chemistry packages.
+- Q# ergonomic improvements:parentheses on the left-hand side of a curried call are now optional, eliminating the need for parentheses around subsequent calls; parentheses around operation and function type are now optional; parentheses around conditions and in other statement headers are now optional.
```

Comment:
The optional parentheses for curried calls was in the November release.

For parentheses in statements, it's a little complicated because for statements that take an expression, they're optional (because we can't prevent an expression from being wrapped in parentheses). But for other statements like `for`, `use` and `borrow`, parentheses are actually deprecated and will be removed. Maybe some wording like this would work:

```suggestion
- Parentheses are now optional for operation and function types and `if`, `elif`, `while` and `until` statements. Parentheses for `for`, `use` and `borrow` statements have been deprecated.
```